src/pykiso/test_coordinator/test_suite.py

```diff
@@ -378,16 +378,15 @@ def check_suite_setup_failed(
     ) -> None:
         """Check if the suite setup has failed and store failed suite id.
         Search in the global unittest result object, which save all the results
-        of the tests performed up to that point, for a BasicTestSuiteSetup tests
-        which has failed. If the suite setup has failed store the suite id.
+        of the tests performed up to that point, if the suite setup that runned
+        has failed then store the suite id.
 
         :param test: test to check
         :param result: unittest result object
         """
         if isinstance(test, BasicTestSuiteSetup):
-            for suite_type, _ in result.failures:
-                if isinstance(suite_type, BasicTestSuiteSetup):
-                    self.failed_suite_setups.add(test.test_suite_id)
+            if result.error_occurred:
```

Comment:
Im wondering why the old code doesnt work anymore. did we update ab deps ?
anyways you approach also looks ok

Reply:
What do you mean?

Reply:
But yes, the previous check was removed. Is there a reason for that?

Reply:
Somehow Im sure that the old code has worked and i dont get it why its now broken Oo

Reply:
The previous check was removed because my new check add the same suite id than looking in result failures, because when a test is added in `result.failures` it also put `result.error_occurred` to true. : https://github.com/eclipse/kiso-testing/blob/master/src/pykiso/test_result/text_result.py#L211-L221
One advantage is that if we have multiple suite it will not skip the second one if the first suite setup fails.

Reply:
The old code seems to be working if the suite setup has an assertion error but not if another type of error raised.